content/de/nondiscrimination.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Nicht-Diskriminierungserklärung und Regel
+slug: nondiscrimination
+top_graphic: 4
+date: 2015-06-18
+---
+
```

Comment:
The privacy policy (privacy.md) and the Non-Discrimination Statement and Policy (nondiscrimination.md) may be translated, but with a note on top saying that the English version is canonical.
https://github.com/letsencrypt/website/blob/master/TRANSLATION.md